repositories/analytics_views/data_model_fields.sql

```diff
@@ -0,0 +1,16 @@
+DROP VIEW IF EXISTS analytics.data_model_fields;
+
+CREATE VIEW
+      analytics.data_model_fields
+WITH
+      (security_invoker = false) AS
+SELECT
+      id,
+      table_id,
+      name,
+type,
+nullable,
+description,
+is_enum
```

Comment:
format issue

Reply:
oopsies

Reply:
ok actually it's not "oopsies", it's just that "type" is a postgres keyword so the sql formatter tries to apply its own format (it's not a reserved keyword though mind you, so it's fine to use it as a field name https://www.postgresql.org/docs/current/sql-keywords-appendix.html) 
the alternative is to use
```
SELECT
      id,
      table_id,
      name,
      nullable,
      description,
      is_enum,
      "type" AS "type"
```
I'm just not sure if it's worth it